vars/Debian-10.yml

```diff
@@ -1,12 +1,11 @@
 ---
-__postgresql_version: "11"
+__postgresql_version: "{{ postgresql_version | default('11') }}"
 __postgresql_data_dir: "/var/lib/postgresql/{{ __postgresql_version }}/main"
 __postgresql_bin_path: "/usr/lib/postgresql/{{ __postgresql_version }}/bin"
 __postgresql_config_path: "/etc/postgresql/{{ __postgresql_version }}/main"
-__postgresql_daemon: "postgresql@{{ postgresql_version }}-main"
+__postgresql_daemon: "postgresql@{{ __postgresql_version }}-main"
 __postgresql_packages:
-  - postgresql
-  - postgresql-contrib
+  - "postgresql-{{ __postgresql_version }}"
```

Comment:
...instead, you could change this to `{{ postgresql_version | default('11') }}`.

That does lead to a tiny bit of code duplication, but seems better than changing the structure of the vars (which could behave funny if changed via the `postgresql_version` anyways).

This is also getting rid of `postgresql-contrib`—is that an intentional change?

Reply:
To my understanding, at least as of PG10, the contrib package is part of PG package as per :

https://packages.debian.org/buster/postgresql-contrib 

and if you look at the file list, it contains only a changelog and copyright info files. 

https://packages.debian.org/buster/all/postgresql-contrib/filelist

I can revert the change or make it install corresponding version of the contrib. 

What do you think?